src/HubPortal.sol

```diff
@@ -83,6 +91,40 @@ contract HubPortal is IHubPortal, Portal {
         emit RegistrarListStatusSent(destinationChainId_, messageId_, listName_, account_, status_);
     }
 
+    /// @inheritdoc IHubPortal
+    function sendMerkleRoots(bytes32 refundAddress_) external payable returns (uint64 sequence_) {
+        bytes32 destinationToken_ = destinationMToken[_SOLANA_WORMHOLE_CHAIN_ID];
+        IMerkleTreeBuilder merkleTreeBuilder_ = IMerkleTreeBuilder(merkleTreeBuilder);
+        bytes32 earnersMerkleRoot = merkleTreeBuilder_.getRoot(_SOLANA_EARNER_LIST);
```

Comment:
undescores are missing in lines 98,99? let's revisit coding style practices next time, too many underscores everywhere.